code/drv_ea/src/wattrex_driver_ea/drv_ea.py

```diff
@@ -4,286 +4,326 @@
+        self.__tx_chan.send_data(msg)
 
-    def get_data(self, channel: int = 1) -> DrvEaDataC:
-        '''Read the device data.
-        Args:
-            - None.
-        Returns:
-            - (DrvEaDataC): Returns the device data.
-        Raises:
-            - None.
-        '''
-        if (channel<1 or channel>2) or ('2384' not in self.__properties.model and channel != 1):
-            log.error("Try to apply command to a channel doesn´t exist, "+
-                    "will apply to the only available")
-            raise ValueError
-        current = 0
-        voltage = 0
-        power = 0
-        status = DrvPwrStatusC(DrvPwrStatusE.COMM_ERROR)
-
-        read_all = self.device_handler.send_and_read(f'MEASure:ARRay? (@{channel})')
-        read_all = read_all[0].split(',')
-        if len(read_all) == 0 and read_all[0] is None:
-            status = DrvPwrStatusC(DrvPwrStatusE.COMM_ERROR)
-        else:
-            voltage = int(self.device_handler.decode_numbers(read_all[0]) * _CONSTANTS.MILI_UNITS)
-            current = int(self.device_handler.decode_numbers(read_all[1]) * _CONSTANTS.MILI_UNITS)
-            power = int(self.device_handler.decode_numbers(read_all[2]) * _CONSTANTS.MILI_UNITS)
-        res : DrvEaDataC
-        if channel == 1:
-            self.__actual_data = DrvEaDataC(mode = self.__actual_data.mode, \
-                                            status = status, \
-                                            voltage = voltage, \
-                                            current = current, \
-                                            power = power)
-            res = self.__actual_data
-        else:
-            self.__actual_data2 = DrvEaDataC(mode = self.__actual_data.mode, \
-                                            status = status, \
-                                            voltage = voltage, \
-                                            current = current, \
-                                            power = power)
-            res = self.__actual_data2
-        return res
+        msg = DrvScpiCmdDataC(data_type = DrvScpiCmdTypeE.WRITE,
+                port = self.__port, payload = _ScpiCmds.OUTPUT_ON.value)
+        self.__tx_chan.send_data(msg)
 
 
-    def get_properties(self) -> DrvEaPropertiesC:
-        '''Read the device properties.
+    def disable(self) -> None:
+        ''' Set the device in standby mode.
         Args:
-            - None.
+            - None
         Returns:
-            - (DrvEaPropertiesC): Returns the device properties.
+            - None
         Raises:
-            - None.
+            - None 
         '''
-        return self.__read_device_properties()
+        msg = DrvScpiCmdDataC(data_type = DrvScpiCmdTypeE.WRITE, \
+                                port = self.__port, \
+                                payload = _ScpiCmds.OUTPUT_OFF.value)
+        self.__tx_chan.send_data(msg)
+        self.last_data.mode = DrvBasePwrModeE.WAIT

```

Comment:
Check if command is applied

Reply:
Now ask for output status and set eait if its off